App/localization/languages/it.json

```diff
@@ -0,0 +1,90 @@
+{
+  "close_button": "Chiudi",
+  "error_screen_common_sorry": "Mi spiace!\n",
+  "error_screen_error_cannot_load_cigarettes": "Non possiamo caricare le tue sigarette.",
+  "error_screen_choose_other_location": "Seleziona un'altra posizione",
+  "error_screen_error_description": "C'è un problema con il nostro database, o non c'è nessuna Stazione di Monitoraggio Aria vicino a te. Riprova più tardi!",
+  "error_screen_error_message": "Errore: {{errorText}}",
+  "frequency_daily": "quotidianamente",
+  "frequency_weekly": "settimanalmente",
+  "frequency_monthly": "mensilmente",
+  "home_station_too_far_message": "Non possiamo trovare nessuna stazione vicino a te.\nA questa distanza i risultati possono essere non accurati.",
+  "home_beta_not_accurate": "I numeri non sono accurati al 100%.",
+  "home_share_title": "Lo sapevi che potresti fumare fino a 20 sigarette al giorno, solo per il fatto che vivi in una grande città?",
+  "home_share_message": "Shoot! Ho 'fumato' {{cigarettes}} sigarette oggi per aver respirato aria urbana. E tu? Guarda qui: https://shootismoke.github.io",
+  "home_header_air_quality_station_distance": "Stazione Qualità Aria: {{distanceToStation}}{{distanceUnit}} lontana",
+  "home_header_from_search": "da ricerca",
+  "home_header_change_location": "Cambia Posizione",
+  "home_btn_why_is_station_so_far": "Perchè la stazione è così lontana?",
+  "home_btn_see_detailed_info": "Mostra informazioni dettagliate",
+  "home_btn_see_how_it_works": "Guarda come funziona",
+  "home_btn_more_details": "Maggiori dettagli",
+  "home_btn_faq_about": "Faq/About",
+  "home_btn_share": "Condividi",
+  "home_common_you_smoke": "Fumi",
+  "home_common_you_d_smoke": "Avevi fumato",
+  "home_common_you_ll_smoke": "Fumerai",
+  "home_common_you_smoked": "Hai fumato",
+  "home_common_oh": "Oh",
+  "home_common_cigarette": "Sigaretta",
+  "home_common_cigarettes": "Sigarette",
+  "home_smoked_cigarette_title": "{{swearWord}}! {{presentPast}}\n<{{cigarettes}} {{singularPlural}}>",
+  "home_smoked_there": "there today",
+  "home_wait_more_days": "{{days}} altri giorni",
+  "home_wait_until_results": "per \nsmostrare i tuoi risultati",
+  "home_swear_word_dang": "Dang",
+  "home_swear_word_shoot": "Shoot",
+  "home_swear_word_darn": "Darn",
+  "home_swear_word_geez": "Geez",
+  "home_swear_word_omg": "OMG",
+  "home_swear_word_crap": "Crap",
```

Comment:
Do you have some ideas of funny interjections in Italian we could put here? (no swear words though! to keep it kid-friendly)

Reply:
Yep, I have some ideas on this, I'm working and testing it with next PR. Thanks for the semantic term and for the great project

Reply:
Completed italian translation